src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/VSCodeHelper/VSCodeInstances.cs

```diff
@@ -145,12 +145,12 @@ public static void LoadVSCodeInstances()
                 using var vsCodeIconBitmap = vsCodeIcon.ToBitmap();
 
                 // Workspace
-                using var folderIcon = (Bitmap)Image.FromFile(Path.Join(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Images//folder.png"));
+                using var folderIcon = (Bitmap)Image.FromFile(Path.Join(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Images//folder.light.png"));
```

Comment:
When I compare to the old icon, this looks somewhat worse:
![image](https://github.com/microsoft/PowerToys/assets/26118718/00c645f2-50a4-4c6b-96a5-b54ec5d26f0c)
![image](https://github.com/microsoft/PowerToys/assets/26118718/30cd93bd-6967-45c5-bf91-ecc0e69a6e60)
Comparing to the old one:
![image](https://github.com/microsoft/PowerToys/assets/26118718/c11b1c75-9514-4e60-93a8-58dc6c80a255)

Reply:
@jaimecbernardo Thanks! I have reverted back to colored icons, but I have updated them with assets used by W11 and decreased the dimensions (from 240px to 48px) to be similiar to all the other icon plugins.

![image](https://github.com/microsoft/PowerToys/assets/9866362/ab5604fb-e0f9-4325-bdc3-ebb6e2afc19c)